models/event-videos.json

```diff
@@ -56,6 +60,10 @@
         "example": "https://www.youtube.com/watch?v=cEItmb_a20D",
         "description": "The unique url of the video. Currently only Youtube and Vimeo sources are allowed."
       },
+      "embedUrl": {
+        "type": "string",
+        "example": "https://www.youtube.com/embed/cEItmb_a20D"
```

Comment:
Nit: A `description` could help to give some more context about what the purpose of this property is. (Like the other properties also have)

Reply:
Good point! I also added the `format` property and defined it to be an `url`.